### ISSUE TITLE
Add a close-button to the window to minimize Clipdinger

### ISSUE THREAD

**Issue by humdingerb** [humdingerb/clipdinger#31]
Topic says it all.

**Comment by humdingerb** [comment]
I'm not sure that's a good idea WRT to consistency around the system. Normally the close button in the window tab quits the application, if it's the last or main window. Which is the case with Clipdinger.
I think ESC, ALT+W, or double-clicking the tab are convenient enough without introducing possible confusion of close/minimize/quit.

**Comment by humdingerb** [comment]
> Your own QuickLaunch has a close button, and it doesn't "quits" the app, just "hides" it in Deskbar. (Hoping here you don't decide to remove that one! :-P).

Not true. QuickLaunch does actually quit. :)

Anyway, I came around to your way of thinking about that close button. However - and that may be the reason I removed it in the first place - I don't know how to override the close button. When pressed, it seems to invoke QuitRequested() directly, which leads to quitting the app.

I have now introduced a flag that has to be set to 'true' before actually quitting. Maybe not totally elegant, but until I can teach the window close-button to send a message instead of quitting directly...
